docs/dev-tools/zombienet/overview.md

```diff
@@ -387,7 +387,7 @@ Through the keyword `settings`, it's possible to define the general settings for
 - `image_pull_policy?` ++"string"++ - image pull policy to use in the network. Possible values are `Always`, `IfNotPresent`, and `Never`
 - `local_ip?` ++"string"++ - IP used for exposing local services (rpc/metrics/monitors). Defaults to `"127.0.0.1"`
 - `global_delay_network_global_settings?` ++"number"++ - delay in seconds to apply to the network
-- `node_verifier?` ++"string"++ - allow managing how to verify node readiness or disable by using `None`. Possible values are `None` and `Metric`. Defaults to `Metric`
+- `node_verifier?` ++"string"++ - specify how to verify node readniess or deactivate by using `None`. Possible values are `None` and `Metric`. Defaults to `Metric`
```

Comment:
I'm a bit puzzled by "deactivate" here. Did you set that as a rule? @dawnkelly09

Reply:
readniess - misspelled

Reply:
Readiness is my fault, I'll go fix it. 

@nhussein11 Yes, actually. Disabled is not recommended for inclusive language. This comes from the base Google Dev Docs styling.